src/Elcodi/Admin/MediaBundle/Resources/views/Module/_images-form-field.html.twig

```diff
@@ -32,7 +32,10 @@
                     </a>
                     <ul class="thumbnail-actions">
                         <li>
-                            <a href="{{ url('admin_image_delete', { id: image.id }) }}" class="icon-trash-o"></a>
+                            <a
+                                href="{{ url('admin_image_delete', { id: image.id }) }}{% if redirect_url is defined %}{{ '?redirect-path=' ~ redirect_url }}{% endif %}"
```

Comment:
Sorry, `redirect_url:` should be `redirect_path` now. Shouldn't we replace every other `redirect_url` to be consistent?